src/web/app/components/question-responses/grq-rgq-view-responses/grq-rgq-view-responses.component.ts

```diff
@@ -68,31 +69,35 @@ export class GrqRgqViewResponsesComponent extends ResponsesInstructorCommentsBas
   private filterResponses(): void {
     this.responsesToShow = {};
     this.teamsToUsers = {};
+    this.usersToTeams = {};
     this.teamExpanded = {};
     this.userExpanded = {};
     for (const question of this.responses) {
       for (const response of question.allResponses) {
         if (this.isGrq) {
           this.teamsToUsers[response.giverTeam] = this.teamsToUsers[response.giverTeam] || [];
+          this.usersToTeams[response.giver] = this.usersToTeams[response.giver] || [];
           if (this.teamsToUsers[response.giverTeam].indexOf(response.giver) === -1) {
             this.teamsToUsers[response.giverTeam].push(response.giver);
+            this.usersToTeams[response.giver].push(`(${response.giverTeam})`);
             this.teamExpanded[response.giverTeam] = false;
           }
           this.userExpanded[response.giver] = false;
-        } else {
-          if (!response.recipientTeam) {
-            // Recipient is team
-            this.teamsToUsers[response.recipient] = this.teamsToUsers[response.recipient] || [];
-            if (this.teamsToUsers[response.recipient].indexOf(response.recipient) === -1) {
-              this.teamsToUsers[response.recipient].push(response.recipient);
-              this.teamExpanded[response.recipient] = false;
-            }
-            this.userExpanded[response.recipient] = false;
-            continue;
+        } else if (!response.recipientTeam) {
```

Comment:
Do you accidentally change the `if` statement?

Reply:
Nope I did it on purpose. The `if ... continue` logic could just be simplified with a `if else` condition branch imo

Reply:
I don't think the simplification make thing clearer as the previous `if-else` branch make clear distinguish between true/false value of `isGrq`.

Another way to think of this is by looking at this chunk:
https://github.com/TEAMMATES/teammates/blob/bbfcda22de5ddb9c45b51eb38e87a434030b1883/src/web/app/components/question-responses/grq-rgq-view-responses/grq-rgq-view-responses.component.ts#L77-L104

Can you immediately tells what code will be executed if `isGrq` is `false`.

Reply:
Ah okay I get what you mean, the more important distinction is in `isGrq`. I'll revert the changes then :)

Reply:
I reverted the top level branch tree but kept the changes to the nested branch (i.e. use `if else` instead of `if continue`. Let me know if there's any issue with that, thanks!